index.html

```diff
@@ -0,0 +1,32 @@
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+    <header class="header">
+        <img src="assets/icons/menu.svg" alt="Menu">
+    </header>
+    <section>
```

Comment:
- [ ] Kindly consider adding the background image from the figma design as your design doesn't have it and we need to make our design identical to the figma design.
your design
![re](https://user-images.githubusercontent.com/96683843/227307214-b31dc037-6393-48f1-82f7-8a7ce7b5217e.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/227307274-a71f0944-ee4e-40ec-9ebb-4bed75d8e1b5.jpg)